test/e2e/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	k8s.io/apimachinery v0.0.0-20191121015412-41065c7a8c2a
 	k8s.io/client-go v11.0.0+incompatible
 	sigs.k8s.io/cluster-api v0.2.6-0.20191216153338-db9baf9ade0c
-	sigs.k8s.io/cluster-api/test/framework v0.0.0-00010101000000-000000000000
+	sigs.k8s.io/cluster-api/test/framework v0.0.0-20191216153338-db9baf9ade0c
```

Comment:
I'm a little unsure about this change. It seems like you'd have to commit the e2e stuff first and then bump this version in a separate commit since it's referencing a commit hash? Maybe this should be done after the e2e work is finished.